mozilla/fx-private-relay
# MPP-3448 Migration to remove remaining_minutes

This column has not been used since we introduced remaining_seconds and the Django model property remaining_minutes.

Fix MPP-3448
Follow-up to PR #3908
⚠️  ~DO NOT MERGE until PR #3908 is released to production~ Update: PR #3908 tagged on `2023.09.21` and released to production on September 26th, 2023.

How to test:
1. Wait until PR #3908 is deployed to production
2. Rebase the changes to latest main
3. Check that the CI jobs `sqlite migrations test` and `postgres migrations test` are passing
Note: The tests were failing before because the Relay code still had reference to the `deprecated_remaining_minutes` Django field which caused errors when the migration to remove the DB column` remaining_minutes` were applied.
